pulsar-client/src/main/java/com/yahoo/pulsar/client/api/ClientConfiguration.java

```diff
@@ -309,4 +310,22 @@ public long getStatsIntervalSeconds() {
     public void setStatsInterval(long statsInterval, TimeUnit unit) {
         this.statsIntervalSeconds = unit.toSeconds(statsInterval);
     }
+
+    /**
+     * Get configured total allowed concurrent lookup-request.
+     * 
+     * @return
+     */
+    public int getConcurrentLookupRequest() {
+        return concurrentLookupRequest;
+    }
+
+    /**
+     * Configure total allowed concurrent lookup-request to prevent overload on broker.
```

Comment:
We should note here in the javadoc that the default is 5000 and that one should not worry to change this unless you plan to produce/subscribe on 1000s of topics from the same client instance.